domoticz.sh

```diff
@@ -15,12 +15,14 @@ PATH=/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/bin
 DESC="Domoticz Home Automation System"
 NAME=domoticz
 USERNAME=pi
-PIDFILE=/var/run/$NAME.pid
+# Use this PID file to run as regular user
+PIDFILE=/tmp/$NAME-$USERNAME.pid
+
 SCRIPTNAME=/etc/init.d/$NAME
 
 DAEMON=/home/$USERNAME/domoticz/$NAME
 DAEMON_ARGS="-daemon"
-#DAEMON_ARGS="$DAEMON_ARGS -daemonname $NAME -pidfile $PIDFILE"
+DAEMON_ARGS="$DAEMON_ARGS -daemonname $NAME -pidfile $PIDFILE"
```

Comment:
The PID file is already specified as argument of the start-stop-daemon command... could be removed here. Maybe this parameter could be changed to include the 'chuid' ?